homeassistant/components/tahoma/light.py

```diff
@@ -0,0 +1,133 @@
+"""Tahoma light platform that implements dimmable tahoma lights."""
+from datetime import timedelta
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_EFFECT,
+    SUPPORT_BRIGHTNESS,
+    SUPPORT_EFFECT,
+    LightEntity,
+)
+from homeassistant.const import STATE_OFF, STATE_ON, STATE_UNKNOWN
+
+from . import DOMAIN as TAHOMA_DOMAIN, TahomaDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=30)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the Tahoma light platform."""
+    if discovery_info is None:
+        return
+    controller = hass.data[TAHOMA_DOMAIN]["controller"]
+    devices = []
+
+    for device in hass.data[TAHOMA_DOMAIN]["devices"]["light"]:
+        devices.append(TahomaLight(device, controller))
+
+    async_add_entities(devices, True)
+
+
+class TahomaLight(TahomaDevice, LightEntity):
+    """Representation of a Tahome light."""
+
+    def __init__(self, tahoma_device, controller):
+        """Initialize the device."""
+        super().__init__(tahoma_device, controller)
+        self._skip_update = False
+        self._effect = None
+        self._state = STATE_UNKNOWN
+        self._brightness = 0
+        if self.tahoma_device.type == "io:DimmableLightIOComponent":
+            self._type = "io"
+            self._unique_id = self.tahoma_device.url
+            self.update()
+
+    @property
+    def brightness(self) -> int:
+        """Return the brightness of this light between 0..255."""
+        _LOGGER.debug("[THM] Called to get brightness %s", self._brightness)
+        return int(self._brightness * (255 / 100))
+
+    @property
+    def is_on(self) -> bool:
+        """Return true if light is on."""
+        _LOGGER.debug("[THM] Called to check is on %s", self._state)
+        return self._state == STATE_ON
+
+    @property
+    def supported_features(self) -> int:
+        """Flag supported features."""
+        return SUPPORT_BRIGHTNESS | SUPPORT_EFFECT
+
+    async def async_turn_on(self, **kwargs) -> None:
+        """Turn the light on."""
+        _LOGGER.debug("[THM] Called to turn on (%s, %s)", kwargs, self._brightness)
```

Comment:
```suggestion
        _LOGGER.debug("Called to turn on (%s, %s)", kwargs, self._brightness)
```

The name of the module is already logged